README.md

```diff
@@ -5,130 +5,110 @@
 ![tests](https://github.com/gemini-hlsw/Scheduler/actions/workflows/pytest.yml/badge.svg)
 [![codecov](https://codecov.io/gh/gemini-hlsw/scheduler/branch/main/graph/badge.svg?token=15CBFMK3KP)](https://codecov.io/gh/gemini-hlsw/scheduler)
 
-This is the automated Scheduler for Gemini Observatory, part of the GPP project.
+## Project Overview
 
-For the list of dependencies check: `requirements.txt`.
+The Scheduler is an automated tool for the Gemini Observatory, designed to generate and optimize observation schedules as part of the GPP project. It uses the [lucupy](https://github.com/gemini-hlsw/lucupy) library for modeling and supports both standalone and service modes, as well as a Jupyter-based UI for experimentation.
 
-## How to Install (Local Development)
+## Features
 
-**Note:** These instructions assume you are using Mac OS X or Linux.
+- Automated scheduling of observations
+- GraphQL API for integration with Gemini Program Platform 
+- Jupyter notebooks for interactive exploration
+- Docker support for easy deployment
 
-### Download the project source:
+## Environment Variables
 
-Fork the project and then clone into your desired directory.
+Before running the scheduler, set the following environment variables:
 
-**Optional step:** You may wish to also fork and clone the [lucupy](https://github.com/gemini-hlsw/lucupy) repository, which is the package that contains the model for this project. Otherwise the package dependency will be installed from pypi.
-
-### Create the project environment:
-
-You should create a set of environment variables before running the scheduler, one option is adding the following lines to your `~/.bash_profile` or equivalent:
+- `PYTHONPATH`: Should include the project base path.
+- `REDISCLOUD_URL`: Redis connection string (contact project staff for credentials).
+- `APP_VERSION`: Application version (e.g., `dev`).
 
+Example:
 ```shell
-export PYTHONPATH=$PYTHONPATH:{path-to-project-base}
-export REDISCLOUD_URL redis://<USER>:<PASSWORD>@redis-12725.c261.us-east-1-4.ec2.cloud.redislabs.com:12725
-export APP_VERSION dev
-```
+export PYTHONPATH=$PYTHONPATH:/path/to/scheduler
+export REDISCLOUD_URL=redis://<USER>:<PASSWORD>@redis-12725.c261.us-east-1-4.ec2.cloud.redislabs.com:12725
```

Comment:
Even if the user and password are not in the URL (thus not allowing the connection) having the URL public does not stop people from trying. I think is better if we keep it more hidden.